projects/js-packages/charts/src/components/pie-chart/pie-chart.tsx

```diff
@@ -27,20 +44,23 @@ interface PieChartProps extends BaseChartProps< DataPointPercentage[] > {
  */
 const PieChart = ( {
 	data,
-	width = 500, //TODO: replace when making the components responsive
-	height = 500, //TODO: replace when making the components responsive
+	size = 500, //TODO: replace when making the components responsive
+	thickness = 1,
 	withTooltips = false,
-	innerRadius = 0,
```

Comment:
I hope it isn't too late to remove this property 😬

Reply:
Good from the Jetpack side at least 😄